### ISSUE TITLE
Undefined Method WebhookTesting

### ISSUE THREAD

**Issue by deandex** [braintree/braintree_php#214]
### General information

* SDK/Library version: 3.30.0
* Environment: both
* Language, language version, and OS: PHP on Ubuntu 16.04

### Issue description

```php
$braintreeGateway = new \Braintree\Gateway([
    'environment' => $gatewayConfig['environment'],
    'merchantId'  => $gatewayConfig['merchant_id'],
    'publicKey'   => $gatewayConfig['public_key'],
    'privateKey'  => $gatewayConfig['private_key']
]);

if ($testing) {
    $sampleNotification = $braintreeGateway->webhookTesting()->sampleNotification(
        \Braintree_WebhookNotification::SUBSCRIPTION_CHARGED_SUCCESSFULLY,
        'dummy_id'
    );

    $bt_signature = $sampleNotification['bt_signature'];
    $bt_payload = $sampleNotification['bt_payload'];
}
```

I got undefined method issue on this **$braintreeGateway->webhookTesting()**, seems WebhookTesting not extend to Base class.
Because i see it was updated in documentation [Webhook Testing](https://developers.braintreepayments.com/guides/webhooks/testing-go-live/php)
Or i am wrong on implement it. Please advice

**Comment by crookedneighbor** [comment]
Thanks for the report. We'll have this fixed up in the next release.

**Comment by jackellenberger** [comment]
Hey @deandex, sorry to keep you waiting - this issue with WebhookTesting was resolved in [our latest release, 3.30.0](https://github.com/braintree/braintree_php/blob/master/CHANGELOG.md#3310). Thanks for your patience!
